VideoLocker/src/main/java/org/edx/mobile/module/notification/EdxParsePushBroadcastReceiver.java

```diff
@@ -32,101 +33,129 @@
     //UI and animation time need to get from production team.
     private boolean enableFlyinWindow = false;
 
-    public void onReceive(android.content.Context context, android.content.Intent intent) {
-        super.onReceive(context,intent);
-     }
+    public void onReceive(Context context, Intent intent) {
+        super.onReceive(context, intent);
+    }
 
 
-    protected void onPushReceive(android.content.Context context, android.content.Intent intent) {
-        try{
+    protected void onPushReceive(Context context, Intent intent) {
+        try {
             CourseUpdateNotificationPayload payload = ParseHandleHelper.extractPayload(intent);
-            if ( payload != null && ParseHandleHelper.hasNotificationHash(context, payload.getIdentifier()) )
+            if (payload != null && ParseHandleHelper.hasNotificationHash(context, payload.getIdentifier()))
                 return;
-            if( enableFlyinWindow ) {
+            if (enableFlyinWindow) {
                 Intent intent2 = new Intent(context, NotificationFlyinService.class);
                 intent2.putExtra("com.parse.Data", intent.getExtras().getString("com.parse.Data"));
                 context.startService(intent2);
             }
-        }catch (Exception ex){
+        } catch (Exception ex) {
```

Comment:
I think we should get rid of this generic catch block. The only method that might throw an unchecked exception here is `extractPayload()`, but it already has a generic catch block (which should probably also be converted to catching only `JsonSyntaxException`).